docs/src/content/en/docs/deployment/monorepo.mdx

```diff
@@ -44,23 +58,22 @@ api/
 
 ## Deployment configuration
 
-The image below shows how to select `apps/api` as the project root when deploying to [Mastra Cloud](./mastra-cloud/overview). While the interface may differ between providers, the configuration remains the same.
+The image below shows how to select `apps/api` as the project root when deploying to [Mastra Cloud](/docs/v1/mastra-cloud/overview). While the interface may differ between providers, the configuration remains the same.
 
 ![Deployment configuration](/img/monorepo/monorepo-mastra-cloud.jpg)
 
+
+:::info
+Make sure the correct package (e.g. `apps/api`) is selected as the deploy target. Selecting the wrong project root is a common deployment issue in monorepos.
+:::
+
 ## Dependency management
 
 In a monorepo, keep dependencies consistent to avoid version conflicts and build errors.
 
 - Use a **single lockfile** at the project root so all packages resolve the same versions.
 - Align versions of **shared libraries** (like Mastra or frameworks) to prevent duplicates.
 
-## Deployment pitfalls
-
-Common issues to watch for when deploying Mastra in a monorepo:
-
-- **Wrong project root**: make sure the correct package (e.g. `apps/api`) is selected as the deploy target.
-
 ## Bundler options
```

Comment:
The `transpilePackages` is no longer necessary, we do it automatically now. Let's remove the whole section for now, we need to add information about `bundler.externals` and how that all works holistically to the docs in another PR